### PR TITLE
[FR] Update BanchoBot

### PR DESCRIPTION
**Pull request by UltayJr**

### CODE REVIEW

File: wiki/BanchoBot/fr.md [hunk 1]
```diff
@@ -124,10 +115,10 @@ BanchoBot: peppy est le développeur principal ainsi que le créateur d'osu! et
 ```
 
 ```
-Tama: Bonjour
+Tama: 你好
 Yuzu: !faq chinese
-BanchoBot: Les utilisateurs chinois s'il vous plaît cliquez sur #chinese Pour entrer dans le canal chinois pour la communication.
-BanchoBot: #osu s’agit d’un canal en anglais seulement. Si vous continuez de parler chinois à #osu après avoir reçu ce message, l’administrateur a le droit d'appliquer des sanctions.
+BanchoBot: #osu 是英文专属频道，如果接获此讯息后继续在 #osu 内以中文交谈，管理员有权利禁言。 (Les utilisateurs chinois s'il vous plaît cliquez sur #chinese Pour entrer dans le canal chinois pour la communication.)
```
Comment (almaloste) [inline wiki/BanchoBot/fr.md]:
```suggestion
BanchoBot: 中文用户请点击 #chinese 以进入中文频道进行交流。 (Les utilisateurs chinois s'il vous plaît cliquez sur #chinese Pour entrer dans le canal chinois pour la communication.)
```

File: wiki/BanchoBot/fr.md [hunk 2]
```diff
@@ -124,10 +115,10 @@ BanchoBot: peppy est le développeur principal ainsi que le créateur d'osu! et
...
+BanchoBot: #osu 是英文专属频道，如果接获此讯息后继续在 #osu 内以中文交谈，管理员有权利禁言。 (#osu s’agit d’un canal en anglais seulement. Si vous continuez de parler chinois à #osu après avoir reçu ce message, l’administrateur a le droit d'appliquer des sanctions.)
```
Comment (almaloste) [inline wiki/BanchoBot/fr.md]:
```suggestion
BanchoBot: #osu 是英文专属频道，如果接获此讯息后继续在 #osu 内以中文交谈，管理员有权利禁言。 (#osu est un canal en anglais seulement. Si vous continuez de parler chinois dans #osu après avoir reçu ce message, l’administrateur a le droit d'appliquer des sanctions.)
```

File: wiki/BanchoBot/fr.md [hunk 3]
```diff
@@ -151,19 +144,3 @@ Envoie un rapport à [L'équipe de Modération](/wiki/Global_Moderation_Team) ou
 pippi: !report flyte spamming dans #japanese
```
Comment (almaloste) [inline wiki/BanchoBot/fr.md]:
```suggestion
pippi: !report flyte spamming in #japanese
```

File: wiki/BanchoBot/fr.md [hunk 4]
```diff
@@ -151,19 +144,3 @@ Envoie un rapport à [L'équipe de Modération](/wiki/Global_Moderation_Team) ou
...
 BanchoBot: Les modérateurs du tchat ont été alertés. Merci de votre aide.
```
Comment (almaloste) [inline wiki/BanchoBot/fr.md]:
```suggestion
BanchoBot: Chat moderators have been alerted. Thanks for your help.
```

File: wiki/BanchoBot/fr.md (hunk 3 above)
Comment (UltayJr) [reply wiki/BanchoBot/fr.md]: j'avais même pas fait gaffe, c'est la traduction de la page d'avant x), le Google Traduction était trop puissant pour Doryan apparemment ^^.

**Comment by UltayJr** [comment]
Some original translation errors, fixed now !

File: wiki/BanchoBot/fr.md (hunk 3 above)
Comment (almaloste) [reply wiki/BanchoBot/fr.md]: Yep xD

**Comment by cl8n** [comment]
the messages from BanchoBot should be copied from the game, not translated/commentated

**Comment by cl8n** [comment]
applies to every other banchobot response as well, I think EN has the most updated responses for all of them

**Comment by UltayJr** [comment]
Ok, seems good now, is there something else to report ?

File: wiki/BanchoBot/fr.md [hunk 5]
```diff
@@ -120,14 +112,14 @@ Affiche le message de l'entrée. Vous pouvez utiliser l'argument `list` pour aff
...
+BanchoBot: peppy is the lead developer and indeed, the creator of osu! and handles most of the project himself.
 ```
 
 ```
-Tama: Bonjour
+Tama: 你好
 Yuzu: !faq chinese
-BanchoBot: Les utilisateurs chinois s'il vous plaît cliquez sur #chinese Pour entrer dans le canal chinois pour la communication.
-BanchoBot: #osu s’agit d’un canal en anglais seulement. Si vous continuez de parler chinois à #osu après avoir reçu ce message, l’administrateur a le droit d'appliquer des sanctions.
+BanchoBot: #osu 是英文专属频道，如果接获此讯息后继续在 #osu 内以中文交谈，管理员有权利禁言。
```
Comment (almaloste) [inline wiki/BanchoBot/fr.md]: Je maintiens que la 1ère phrase en chinois de bancho bot est celle ci
```suggestion
BanchoBot: 中文用户请点击 #chinese 以进入中文频道进行交流。
```

File: wiki/BanchoBot/fr.md [hunk 6]
```diff
@@ -141,29 +133,15 @@ Tama: ok
 !REPORT <utilisateur> <raison>
 ```
 
+*Afin de savoir ce qui mérite d'être signalé, veuillez vous référer à [Signaler un mauvais comportement](/wiki/Reporting_Bad_Behaviour).*
+
 *Remarque: lorsque vous entrez le nom d'utilisateur, remplacez les espaces par des underscores (`_`).*
 
 Envoie un rapport à [L'équipe de Modération](/wiki/Global_Moderation_Team) ou les [Modérateurs par langues](/wiki/Language_Moderators).
```
Comment (almaloste) [inline wiki/BanchoBot/fr.md]:
```suggestion
Envoie un rapport à [L'équipe de Modération](/wiki/Global_Moderation_Team) ou aux [Modérateurs par langues](/wiki/Language_Moderators).
```

**Comment by UltayJr** [comment]
> 
> 
> missing the note about REQUEST around line 15

It is now fixed, no ?

**Pull request merged**